src/backend/commands/tablecmds.c

```diff
@@ -4519,6 +4522,10 @@ RenameRelation(RenameStmt *stmt)
 	/* Do the work */
 	RenameRelationInternal(relid, stmt->newname, false, is_index_stmt);
 
+	/* Rry to Rename in gp_matview_aux too. */
+	if (is_matview)
```

Comment:
no need `is_matview` var,  if (relkind == RELKIND_MATVIEW) ?

Reply:
`relkind ` wil be changed during loop, seems only for alter index command.
But you'r right, there is no need use another var, stmt is enough for this, stmt->renameType

Reply:
Fixed.